From e096eb0000f67a64e92856a8ff71877a4b57d6d5 Mon Sep 17 00:00:00 2001
From: Fan Chen <fan.chen@mediatek.com>
Date: Thu, 27 Aug 2015 14:56:47 +0800
Subject: [PATCH 1/8] arm64: dts: mediatek: mt8173 PSCI-1.0 support

This patch adds psci comatible properties to support all mandatory
functions of PSCI-1.0, PSCI-0.2 and PSCI-0.1, and system suspend
can be supported in PSCI-1.0.

Change-Id: Ib2db071530f39cecbf2cdc06ae228d997d00b04c
Signed-off-by: Fan Chen <fan.chen@mediatek.com>
---
 arch/arm64/boot/dts/mediatek/mt8173.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/mediatek/mt8173.dtsi b/arch/arm64/boot/dts/mediatek/mt8173.dtsi
index 94ef89e..9661d5a 100644
--- a/arch/arm64/boot/dts/mediatek/mt8173.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt8173.dtsi
@@ -96,7 +96,7 @@
 	};
 
 	psci {
-		compatible = "arm,psci";
+		compatible = "arm,psci-1.0", "arm,psci-0.2", "arm,psci";
 		method = "smc";
 		cpu_suspend   = <0x84000001>;
 		cpu_off	      = <0x84000002>;
-- 
1.9.1

